.identity/02_application_main_auth.tf

```diff
@@ -0,0 +1,17 @@
+resource "azurerm_role_assignment" "main_terraform_subscription" {
+  scope                = data.azurerm_subscription.current.id
+  role_definition_name = "Reader"
+  principal_id         = azuread_service_principal.main.object_id
+}
+
+resource "azurerm_role_assignment" "main_terraform_storage_account_tfstate_app" {
```

Comment:
do we want to remove it and add it only if it's needed?

Reply:
ok, i do it